cfme/cloud/stack.py

```diff
@@ -327,7 +342,7 @@ def get_tags(self):
 
     def wait_for_exists(self):
         """Wait for the row to show up"""
-        view = navigate_to(self, 'All')
+        view = navigate_to(StackCollection, 'All')
```

Comment:
`navigate_to(self.collection, 'All')` should be future-proof.

The sense of collections is to remove registering and using just classes without context.